bridge/github/datakit_github.ml

```diff
@@ -95,6 +95,7 @@ module Commit = struct
 
   type t = { repo: Repo.t; id : string }
 
+  let create repo id = {repo; id = String.trim id }
```

Comment:
I think it would make more sense to reject invalid arguments rather than silently trying to fix them. It would be very confusing for a name to work in some places and not others.